test_aodndata/acorn/test_acorn.py

```diff
@@ -0,0 +1,84 @@
+import os
+from datetime import datetime
+
+from aodncore.pipeline import PipelineFilePublishType
+from aodncore.pipeline.exceptions import InvalidFileNameError
+from aodndata.acorn.handler import AcornHandler
+from aodncore.testlib import HandlerTestCase
+
+TEST_ROOT = os.path.join(os.path.dirname(__file__))
+
+GOOD_NC_FV00 = os.path.join(TEST_ROOT,
+                       'IMOS_ACORN_V_20180501T003000Z_ROT_FV00_1-hour-avg.nc')
```

Comment:
Why do we need to add some actual files on GitHub? Aren't the tests only performed on file names?

Reply:
For tests that only deal with names (strings) this is true, but some real files are useful for other tests like running a file through the whole handler, or for things that need to extract variables etc.

In some cases it might be appropriate to mock out the bits that deal with actual files, but sometimes it's really handy to have a file which you *know* should always pass through with an expected result and a file which you know should fail (and if not, the lack of a failure is considered an error).

For example GOOD_NC might be a file you want to say "this file should always be handled successfully, regardless of how the *code* is changed" and similar for a "this file should always be rejected because of a check or because it was missing some variable that the handler needed to generate the dest path".

Reply:
The content of the file is also tested here, to retrieve the date_create attribute

Reply:
The test on creation_date didn't exist in previous pipeline though. I'm hoping we're not going too far in what is just a migration.

Reply:
well you can just test about anything in unittests. It just testing that the function is behaving fine, which means that in case someone changes this function, well the unittest will fail accordingly. It makes it easier to program something knowing you 're not going to break things expectingly.

It is a migration, but with a total different design in mind